frontend/src/containers/Login/index.tsx

```diff
@@ -131,4 +136,15 @@ class Login extends React.Component<{}, State> {
   }
 }
 
-export default Login
+const mapDispatchToProps = (dispatch: any) => {
+  return {
+    onAuthSuccess: (id_token: string, auth_code: string) => {
+      dispatch(onAuthSuccess(id_token, auth_code))
+    },
```

Comment:
@chloe-jiang You imported `onAuthFailure` from `auth/actions`. Now we need to map the function to props within `mapDispatchToProps`. Follow the format for `onAuthSuccess`